src/tests/eina/eina_test_file.c

```diff
@@ -29,6 +29,7 @@
 
 #ifdef _WIN32
 # include <evil_private.h> /* mkdir */
+# include <windows.h>
```

Comment:
`evil_private.h` is indirectly including `windows.h` already. I would suggest to use `WIN32_LEAN_AND_MEAN` if you want to explicitly keep it here.

Reply:
Done.